website/content/api-docs/secret/aws.mdx

```diff
@@ -585,3 +585,136 @@ $ curl \
   }
 }
 ```
+
+## Create Static Role
```

Comment:
Not sure if this is already planned, but it'd be neat to have a mention on the [docs](https://developer.hashicorp.com/vault/docs/secrets/aws) page that this feature exists! Might be hard to discover in API docs alone.

Reply:
I threw something very basic up just as a pointer.

Reply:
(pinging @yhyakuna in case she wants to take a look)